service/service.py

```diff
@@ -204,7 +204,7 @@ def post(self):
         return message, status.HTTP_201_CREATED, {'Location': location_url}
 
     #------------------------------------------------------------------
-    # QUERY AND LIST WISHLIST
```

Comment:
Why was query removed?

Reply:
I feel "query and list" is a bit confusing. "List" only feels more clear on what the API is doing.

Reply:
I think mentioning query is necessary since the same function supports querying. Since query is an API we had to implement as a part of the assignment, it's better to explicitly call it out.

Reply:
the code was modified, the comment was adjusted since it is for list. It was reviewed by Manuel.

Reply:
But it queries and lists. When there is no specific value input, it lists, and when there are certain fields provided, it queries.

Reply:
I let Manuel decide, since he made the change.

Reply:
I'd still suggest to keep it QUERY and LIST

Reply:
Let's keep QUERY and LIST.